src/AppMain.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;700;800&display=swap');
+
+div {
+  background: #EEF0F1;
+}
+
+.App {
+  text-align: center;
+  font-family: 'Mulish', sans-serif;
+  min-height: 50rem;
+}
+
+.NavBar {
+  width: 100%;
+  height: 5rem;
+  text-align: center;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  flex-direction: row;
+  position: fixed;
+  top: 0;
+  background: #4267B2;
+  color: white;
+  min-width: 50%;
+  white-space: nowrap;
+  font-weight: 800;
+  z-index: 1;
+}
+
+#TeachLA {
+  width: 3.75rem;
+  height: 3.75rem;
+  left: 20px;
+  margin: 1.25rem;
+}
+
+#BiasByUs {
+  font-size: 3rem;
+  margin: 0.5625rem;
+}
+
+#currentCaseStudy {
+  font-size: 1.875rem;
+  margin: 1.25rem;
+  position: relative;
+  top: 0.3125rem;
+}
+
+button {
+  background-color: #EEF0F1;
+  width: 4.063rem;
+  min-width: 4.063rem;
+  height: 4.063rem;
+  border-radius: 50%;
+  border: none;
+  position: relative;
+  margin-left: auto;
+  margin-right: 4.375rem;
+  display: inline-block;
+}
+button:focus {
+  outline: none;
+}
+.triangle {
+  width: 0;
+	height: 0;
+	border-left: 1.25rem solid transparent;
+	border-right: 1.25rem solid transparent;
+  border-top: 1.563rem solid black;
+  position: relative;
+  left: 0.375rem;
+  top: 0.25rem;
+}
+.up {
+  transform: rotate(180deg);
+  top: -0.0625rem;
+}
+
+
+
+
+/* everything below is for component placeholders */
+
+#dropdownPlaceholder {
+  font-size: 1.5rem;
+  background-color: black;
+  color: white; 
+  height: 12.5rem;
+  width: 12.69rem;
+  position: fixed;
+  top: 6.25rem;
+  right: 0;
+}
+
+.Posts {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  position: relative;
+  top: 5rem;
+  z-index: 0;
+}
+p {
+  font-size: 2rem;
+  border: black 0.0625rem solid;
+  /*width: 59.38rem;*/
+  width: 66%;
+  height: 25rem;
+  /*padding: 400px 300px 400px 300px;*/
+  background-color: white;
+  border-radius: 1.25rem;
+}
+
+.ProgressBar {
+  position: fixed;
+  top: 7.5rem;
+  background-color: black;
+  color: white;
+  width: 9.4%;
+  height: 78%;
+  max-width: 8.438rem;
```

Comment:
These numbers seem a bit arbitrary, is there a reason why you picked them?

Reply:
I wanted the progress bar to be proportional to the screen size. On the Figma design, the progress bar is 135 pixels wide and 800 pixels long on a 1440 x 1024 screen, so to width = 135 / 1440 and height = 800 / 1024. My monitor is pretty big and the progress bar looked too wide when the browser was on full screen, so I set the max width to 800 pixels (8.438rem). Of course, all of this is subject to change after we actually create the progress bar component and see how it fits in with the page.

Reply:
Oh okay, gotcha gotcha. If the design is scalable enough, then I think this is fine - should be good for now!